bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/handler/EspMilightHubHandler.java

```diff
@@ -56,6 +58,46 @@
  */
 @NonNullByDefault
 public class EspMilightHubHandler extends BaseThingHandler implements MqttConnectionObserver, MqttMessageSubscriber {
+    // these are all constants used in color conversion calcuations.
+    // strings are necessary to prevent floating point loss of precision
+    private static final BigDecimal BIG_DECIMAL_THOUSAND = new BigDecimal(1000);
```

Comment:
Should these get moved into this file?

https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/EspMilightHubBindingConstants.java

Reply:
These constants are private and so only used in this class. So this is ok for me. Moving them in the class dedicated for constants would also be ok for me. So the 2 options are fine for me and I have no clear recommendation there.

Reply:
Let's leave as is then. Resolving comment.